app/routes/crate/versions.js

```diff
@@ -0,0 +1,12 @@
+import Route from '@ember/routing/route';
+import { waitForPromise } from '@ember/test-waiters';
+
+export default class VersionsRoute extends Route {
+  setupController(controller) {
+    super.setupController(...arguments);
+    let crate = this.modelFor('crate');
+    controller.set('crate', crate);
+    // TODO: Add error handling
+    waitForPromise(crate.loadVersionsTask.perform());
```

Comment:
I'm wondering if it makes more sense to call this task in the `model()` or at least `afterModel()` hooks, so that the data is fully loaded before the route displays anything. I don't know why it was originally implemented this way.

I guess this is a non-blocking concern though, since the code previously used the same pattern 🤔

Reply:
Although implementing it this way could potentially be more complex, it would also be more responsive, as it could display all other elements (layout, crate header) before the versions are ready (a loading indicator for this hasn't been implemented yet).

Reply:
But yeah, I mostly implemented it following the previous pattern. But I guess putting this either in `model()` or `afterModel()` should also work for non-blocking purposes, just note not to await for it.